package.json

```diff
@@ -46,6 +47,7 @@
     "gulp-rename": "^1.2.0",
     "gulp-uglify": "^1.1.0",
     "istanbul": "^0.3.5",
+    "jayson": "^1.2.2",
```

Comment:
Looks like this is also used in the `src` directory, so I think it should be a non-dev dependency (which means we also need to shrinkwrap).